recording_server/src/utils/system-monitor.ts

```
@@ -0,0 +1,260 @@
+        this.botUuid = botUuid
+        this.secret = secret
+        const pathManager = PathManager.getInstance(botUuid, secret)
+        this.logPath = pathManager.getMachineLogPath()
+    }
+
+    async getSystemCpuUsage(): Promise<number> {
+        try {
+            const { stdout } = await execAsync(
+                "top -bn1 | grep 'Cpu(s)' | awk '{print $2}' | cut -d'%' -f1",
+            )
+            return parseFloat(stdout.trim()) || 0
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)